docs/template_extension/extension.cpp

```diff
@@ -108,6 +108,11 @@ InferenceEngine::ILayerImpl::Ptr Extension::getImplementation(const std::shared_
 }
 //! [extension:getImplementation]
 
+// Exported function
+INFERENCE_EXTENSION_API(void) InferenceEngine::CreateExtensionShared(std::shared_ptr<InferenceEngine::IExtension>& ext) {
+    ext = std::make_shared<Extension>();
+}
```

Comment:
When I commented this function implementation, I have:
```
&strContains(ex.what(), "name: custom_opset. Opset") evaluates to false, where
ex.what() evaluates to bad_weak_ptr
"name: custom_opset. Opset" evaluates to name: custom_opset. Opset
[WARN] 2021-02-20T13:32:32z frontend/onnx_import/src/ops_bridge.cpp 201 Overwriting existing operator: custom_domain.Template:1
[WARN] 2021-02-20T13:32:32z frontend/onnx_import/src/ops_bridge.cpp 201 Overwriting existing operator: custom_domain.FFT:1
/home/sandye51/Documents/Programming/git_repo/dldt/inference-engine/tests/functional/inference_engine/core_threading_tests.cpp:54: Failure
&strContains(ex.what(), "name: custom_opset. Opset") evaluates to false, where
ex.what() evaluates to bad_weak_ptr
"name: custom_opset. Opset" evaluates to name: custom_opset. Opset
[  FAILED  ] CoreThreadingTests.ReadNetwork (100 ms)
```

Looks like compatibility path does not work properly

Reply:
Easy reproduced with `CustomOpsSerializationTest.CustomOpUser_MO` test

Reply:
What binary run this test?